src/strings/PowerFxResources.en-US.resx

```diff
@@ -4075,7 +4075,7 @@
     <comment>This error message shows up when an invalid type is used</comment>
   </data>
   <data name="ErrUDF_ReturnTypeDoesNotMatch" xml:space="preserve">
-    <value>The stated function return type does not match the return type of the function body.</value>
+    <value>The stated function return type '{0}' does not match the return type of the function body '{1}'.</value>
     <comment>This error message shows up when expected return type does not match with actual return type</comment>
```

Comment:
Please add to the comment the description of the placeholders, and an example. Something like `The argument '{0}' and '{1}' will be replaced with data types. For example, "The stated function return type 'Number' does not match the return type of the function body 'Person'"`